content/blog/2023-08-04-out-in-the-open-july-2023.md

```diff
@@ -0,0 +1,41 @@
+og_image: out-in-the-open-2023-07-header.jpg
+type: blog
+---
+
+Das Urheberrecht erfüllt eine wichtige Aufgabe – es schützt geistiges Eigentum. Doch wie steht es um Werke, die von öffentlichen Stellen angefertigt und finanziert werden? Diese enthalten oft Inhalte, die dem Gemeinwohl nützen würden. Dennoch bleiben sie weitestgehend unter Verschluss. Denn von außen ist häufig nicht eindeutig erkennbar, ob ein amtliches Werk dem Urheberrecht unterliegt oder nicht. Stefan Kaufmann fordert [in seiner Kolumne](https://netzpolitik.org/2023/oeffentliches-geld-oeffentliches-gut-wie-das-urheberrechtsgesetz-geaendert-werden-muss/) auf netzpolitik.org: Das Urheberrecht muss geändert werden!
+Die Kolumn zieht u. a. Vergleiche mit den USA, in denen Werke der Bundesbehörden nicht dem Urheberrecht unterliegen. In diesem Zusammenhang interessant: Ein [Bericht über den Correctiv-Datenjournalisten Max Donheiser](https://www.journalist.de/startseite/detail/article/jeder-datenpunkt-eine-geschichte), der von New York nach Deutschland gezogen ist. Dieser stellt im Vergleich fest, dass sich die Datenrecherche hierzulande oftmals schwierig gestaltet: "Hier in Deutschland gibt es viele Regeln, aber wenig Daten".
+
+Doch dass sich der Staat mit Reformen und neuen Gesetzen schwer tut, zeigt sich dieser Tage einmal mehr: Die Umsetzung eines Bundestransparenzgesetzes schiebt die Regierung vorerst auf die lange Bank. Die Eckpunkte für das Gesetz hätten eigentlich schon Ende letzten Jahres stehen sollen. [Ingo Dachwitz sieht darin ein großes Problem](https://netzpolitik.org/2023/plaene-des-innenministeriums-die-ampel-darf-mehr-transparenz-nicht-auf-die-lange-bank-schieben/): Der Zeitverzug sorgt für eine Überschneidung mit dem Ende der aktuellen Legislaturperiode. Dieser Umstand verspricht nichts Gutes – denn mitten im Wahlkampf sinkt die Wahrscheinlichkeit, dass ein so wichtiges wie umstrittenes Gesetz verabschiedet wird.
+
+Dabei täte mehr Transparenz der Bundesregierung und den Ministerien bekanntlich gut. Der Status quo gibt ein düsteres Bild ab: Möchte man wissen, wer in welchen Ministerien die wichtigsten digitalpolitischen Positionen innehat, darf man nicht auf schnelle Antworten hoffen. Dénes Jäger zeichnet [in einem Gastbeitrag](https://netzpolitik.org/2023/transparenz-die-verborgene-digitalpolitik-deutscher-ministerien#netzpolitik-pw) auf netzpolitik.org die mühevolle Suche nach den Verantwortlichkeiten in der Digitalpolitik nach.
+
+Anderswo zeigt man sich gar rückschrittlich: So plant die Stadt Fürth, [die in 2012 in Kraft getretene Informationsfreiheitssatzung abzuschaffen](https://twitter.com/fragdenstaat/status/1682292932101980166?s=20). Der Anwendungsbereich der IFS entfalte "kaum noch praktische Relevanz". Zu der Beschlussvorlage gibt es [bereits mehrere Eingaben](https://stadtrat.fuerth.de/vo0050.asp?__kvonr=61777). Der Antrag wurde [nun zumindest vertagt](https://twitter.com/_stk/status/1684558146810048512?s=20).
+
```

Comment:
```suggestion

Eine gute Nachricht kommt aus Baden-Württemberg. Dort ist vor Kurzem [das neue Open Data-Portal daten.bw](https://www.baden-wuerttemberg.de/de/service/presse/pressemitteilung/pid/oeffentliche-daten-zentral-nutzbar-machen) ans Netz gegangen. Rund 900 Datensätze aus verschiedenen Bereichen standen zum Startschuss der Plattform zur Verfügung. Wie schnell der Bestand wächst, hängt nun natürlich auch [von den potenziellen Datenlieferanten und der Bewerbung des Portals ab](https://twitter.com/datenschatz/status/1682403908998647813?s=20).

```

Reply:
@okfde/codeforde-devs in Slack there are two thumbs up for this PR, so from my point of view, it can be merged